Documentation/DependencyManager.rst

```diff
@@ -16,3 +16,18 @@ install or upgrade some tool as the tool's dependencies conflict with the
 dependencies on another library). It also allows running versions of tools
 that require a PHP version that is higher than the lowest allowed PHP version
 for this project.
+
+.. _using-phive-to-install-phpcov:
+
+Using PHIVE to install `phpunit/phpcov`
+=======================================
+
+To support php version 7.4 and 8.2 in the `tea` extension, we are using PHIVE
+to install `phpunit/phpcov`.
+We need `phpunit/phpcov` in version 10 to support php 8.2.
+Our minimum php version 7.4 would prevent the installation with composer.
```

Comment:
I'd suggest to explain this in more detail to explain the problem, e.g., with this wording:

`To keep things simple, we try do model as many dependencies as possible as Composer dependencies in the `composer.json`. However, sometime a dependency is not available for all PHP versions that we support, or it might conflict with another dependency. One of this cases is PHPCOV, a library we use to collect and transmit code coverage information: The version that works with PHP 8.2 and 8.3 does not work with PHP 7.3. So we have switch this particular dependency from being installed via Composer to being installed via PHIVE.

Reply:
We could also add this explanation:

As we usually only collect code coverage data on CI and not locally, it's usually not necessary to install PHPDOV locally.